src/pages/registration/ContributorsPanel.tsx

```diff
@@ -41,13 +41,34 @@ const ContributorsPanel = () => {
           {({ push, replace }: FieldArrayRenderProps) =>
             publicationContext.type === PublicationType.DEGREE ? (
               <>
-                <Contributors push={push} replace={replace} />
-                <Contributors contributorRole={ContributorRole.SUPERVISOR} push={push} replace={replace} />
+                <Contributors push={push} replace={replace} contributorRoles={[ContributorRole.Creator]} />
+                <Contributors push={push} replace={replace} contributorRoles={[ContributorRole.Supervisor]} />
+                <Contributors
+                  push={push}
+                  replace={replace}
+                  contributorRoles={Object.values(ContributorRole).filter(
+                    (role) => role !== ContributorRole.Creator && role !== ContributorRole.Supervisor
+                  )}
+                />
```

Comment:
Burde kanskje være litt mer luft mellom `Add Author`-knappen og `Contributors`-overskriften?

<img width="640" alt="Skjermbilde 2021-04-22 kl  09 41 35" src="https://user-images.githubusercontent.com/6595982/115675766-2f863300-a34f-11eb-984d-32fdc3a4e9d5.png">

Reply:
ser i skissene nå at det skulle vært annen bakgrunn for hver type